frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -130,18 +125,18 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
         <Split hasGutter isWrappable>
           <SplitItem>
             <Split hasGutter>
-              {isSmallScreen() ? null : (
+              {!isSmallScreen() ? (
```

Comment:
I know this is already existed. But why is `isSmallScreen` a function?

Reply:
Thanks for pointing this out. Changed.